### DISCUSSION TITLE
图片的引用问题

### DISCUSSION THREAD

**Discussion by cyhstart** [hugo-fixit/FixIt#88] [Q&A] [answered]
文章的预览图。和文章中插入的图片应该放在哪儿，老是显示不出来。

**Comment by Lruihao** [comment, marked as answer]
## 文章图片可以存放在三种位置
1. `assets/` 文件夹，例如 [exampleSite/content/about/index.zh-cn.md?plain=1#L25](https://github.com/Lruihao/FixIt/blob/master/exampleSite/content/about/index.zh-cn.md?plain=1#L25)
2. `static/` 文件夹，路径相对于网站根目录
3. 文章文件夹，路径相对于文章，例如
    - [exampleSite/content/posts/theme-documentation-content/index.zh-cn.md?plain=1#L249](https://github.com/Lruihao/FixIt/blob/master/exampleSite/content/posts/theme-documentation-content/index.zh-cn.md?plain=1#L249)
    - [hugo-blog/content/posts/bill-note/index.md?plain=1#L23](https://github.com/Lruihao/hugo-blog/blob/main/content/posts/bill-note/index.md?plain=1#L23)

## 文章的预览图
见 [主题文档 - 内容#前置参数](https://fixit.lruihao.cn/zh-cn/theme-documentation-content/#front-matter) featuredImage 和 featuredImagePreview 参数使用技巧

## 辅助工具参考
> https://lruihao.cn/posts/subl_imgpaste2/

- sublime [imagepaste](https://github.com/robinchenyu/imagepaste)
- vscode [vscode-paste-image](https://github.com/mushanshitiancai/vscode-paste-image)

---
更多关于主题内容的文档可查看 [主题文档 - 内容](https://fixit.lruihao.cn/zh-cn/theme-documentation-content/)

**Reply by xueweiwujxw** [reply]
请问具体怎么配置呢，我的yaml配置如下
```yaml
resources:
- name: "featured-image"
  src: "featured-image.png"
```
这是我的文件路径
![image](https://user-images.githubusercontent.com/45992640/184347540-9a4c51b1-5309-4ed5-af56-d76ccf12c58c.png)
但是没有显示图片

**Reply by Lruihao** [reply]
> 请问具体怎么配置呢，我的yaml配置如下
> 
> ```yaml
> 
> resources:
> 
> - name: "featured-image"
> 
>   src: "featured-image.png"
> 
> ```
> 
> 这是我的文件路径
> 
> ![image](https://user-images.githubusercontent.com/45992640/184347540-9a4c51b1-5309-4ed5-af56-d76ccf12c58c.png)
> 
> 但是没有显示图片

md 文件名称改为 index.en.md 和 index.zh-cn.md 试试

**Reply by xueweiwujxw** [reply]
哦哦，这样是可以的
